### DISCUSSION TITLE
如何配置其他数据库的ssl认证

### DISCUSSION THREAD

**Discussion by lixiaoyong12** [StarRocks/starrocks#33646] [How-tos, Tutorials, Best Practices and Architecture Articles]
源码中的starrocks-ssl-1.0.jar是在哪里？要增加其他数据库认证应该怎么实现

![image](https://github.com/StarRocks/starrocks/assets/34184204/5ff4f45d-4fdc-4b39-9f14-5c905813d7cd)

**Comment by alberttwong** [comment]
questions should be asked in slack.
